src/libraries/Common/src/System/Diagnostics/TraceListenerHelpers.cs

```diff
@@ -9,9 +9,17 @@ internal static partial class TraceListenerHelpers
     {
         private static volatile string? s_processName;
 
-        internal static int GetThreadId()
+        internal static int GetThreadId() => Environment.CurrentManagedThreadId;
```

Comment:
Can we just call `Environment.CurrentManagedThreadId` directly in the few places that it is used?

Reply:
Sure